app/screens/Login/layout.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ImageBackground, Image, View, Text } from 'react-native';
+import imgBackground from '@assets/bc_inicio.png';
+import imgLogo from '@assets/logo.png';
+import RFTextInput from '@app/components/baseComponents/RFTextInput';
+import { white } from '@scss/colors';
+
+import SubmitButton from './components/SubmitButton';
+import { styles } from './styles';
+
+function Login({ handleOnSubmit }) {
+  return (
+    <ImageBackground source={imgBackground} style={styles.imageBackground}>
+      <View style={styles.middleContainer}>
+        <Image source={imgLogo} />
+        <View style={styles.textInputContainer}>
+          <RFTextInput placeholder="Ingrese usuario..." placeholderTextColor={white} />
+          <RFTextInput placeholder="Ingrese contraseña..." placeholderTextColor={white} secureTextEntry />
```

Comment:
maybe you could set `placeholderTextColor: white` value as default in this component